seed/static/seed/js/controllers/menu_controller.js

```diff
@@ -61,13 +61,23 @@ angular.module('BE.seed.controller.menu', [])
       });
       $scope.$on('app_success', function () {
         $rootScope.route_load_error = false;
-      })
+      });
       $scope.$on('organization_list_updated', function () {
         init();
       });
 
       $scope.is_active = function (menu_item) {
-        if (menu_item === $location.path()) {
+        if ($state.current.url.split('/').length > 1) {
+          $location.search('', $state.current.url);
+        }
```

Comment:
Slice current url (if defined) for $location.search to reset location path to reactivate previously clicked tab after "stay on page" option is chosen.